### ISSUE TITLE
Accid encoding is not valid

### ISSUE THREAD

**Issue by martha-thomae** [ECHOES-from-the-Past/GABCtoMEI#1]
The `<accid>` elements cannot be encoded as we are doing now: with `<accid>` inside `<neume>` (as a sibling of the neume components contained within the same "GABC neume"). This is invalid. 

To be valid, `<accid>` needs to be moved at least one level up in the MEI hierarchy, being a sibling of `<neume>` (and `<syl>`). The valid uses of `<accid>` are inside the `<syllable>` element  or outside the `<syllable>` element, but never inside `<neume>`

See the issue reflected in the files: 
* GABC Input: [trial_accid.txt](https://github.com/ECHOES-from-the-Past/GABCtoMEI/blob/main/GABC_infiles/trial_accid.txt)

```
(c4) lis(ixji)
```

* MEI Output: [trial_accid_SQUARE.mei](https://github.com/ECHOES-from-the-Past/GABCtoMEI/blob/main/MEI_outfiles/trial_accid_SQUARE.mei)

```xml
<?xml version="1.0" ?>
<?xml-model href="https://music-encoding.org/schema/5.0/mei-Neumes.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/5.0/mei-Neumes.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0">
  <meiHead>
    <fileDesc>
      <titleStmt>
        <title/>
      </titleStmt>
      <pubStmt/>
    </fileDesc>
  </meiHead>
  <music>
    <body>
      <mdiv xml:id="m-3c615454-8de6-11ee-9d38-f2b2e3f0b574">
        <score xml:id="m-3c615526-8de6-11ee-9d38-f2b2e3f0b574">
          <scoreDef xml:id="m-3c6155d0-8de6-11ee-9d38-f2b2e3f0b574">
            <staffGrp xml:id="m-3c615616-8de6-11ee-9d38-f2b2e3f0b574">
              <staffDef n="1" lines="5" notationtype="neume" xml:id="m-3c61567a-8de6-11ee-9d38-f2b2e3f0b574"/>
            </staffGrp>
          </scoreDef>
          <section xml:id="m-3c6156c0-8de6-11ee-9d38-f2b2e3f0b574">
            <staff n="1" xml:id="m-3c6156fc-8de6-11ee-9d38-f2b2e3f0b574">
              <layer n="1" xml:id="m-3c615742-8de6-11ee-9d38-f2b2e3f0b574">
                <clef shape="C" line="4" xml:id="m-3c615788-8de6-11ee-9d38-f2b2e3f0b574"/>
                <syllable xml:id="m-3c6157c4-8de6-11ee-9d38-f2b2e3f0b574">
                  <syl xml:id="m-3c6157f6-8de6-11ee-9d38-f2b2e3f0b574">lis</syl>
                  <neume xml:id="m-3c615832-8de6-11ee-9d38-f2b2e3f0b574">
                    <accid loc="5" accid="f" xml:id="m-3c615864-8de6-11ee-9d38-f2b2e3f0b574"/>
                    <nc xml:id="m-3c6158a0-8de6-11ee-9d38-f2b2e3f0b574" pname="c" oct="3"/>
                    <nc xml:id="m-3c6158d2-8de6-11ee-9d38-f2b2e3f0b574" pname="b" oct="2"/>
                  </neume>
                </syllable>
              </layer>
            </staff>
          </section>
        </score>
      </mdiv>
    </body>
  </music>
```

**Comment by martha-thomae** [comment]
As an extra comment, the use of `loc` is fine. The attribute `ploc` could be used too, but it is not rendered correctly by Verovio, so I will use `loc` for now as it is also the default behaviour of the converter so far.

**Comment by martha-thomae** [comment]
Fixed in d368cf1 (and 017c368)
